### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by oalders**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,43 +8,34 @@ language: perl
...
     - RELEASE_TESTING=1
 before_install:
-  - git clone git://github.com/travis-perl/helpers ~/travis-perl-helpers
-  - source ~/travis-perl-helpers/init
-  - build-perl
-  - perl -V
-  - build-dist
-  - cd $BUILD_DIR
-install:
-  - cpan-install --deps
```
Comment (genio) [inline .travis.yml:44]: Are you sure we can do without this line? The travis helpers do this?

Comment (genio) [reply .travis.yml:44]: line 44, that is

Comment (oalders) [reply .travis.yml:44]: Yes, see the sample simple configuration: https://github.com/travis-perl/helpers#example-simple-travisyml

**Comment by oalders** [comment]
Thanks @genio!

**Comment by oalders** [comment]
Thanks @genio!

**Pull request merged**